k8s/values.yaml

```diff
@@ -0,0 +1,16 @@
+replicaCount: 1
+
+image:
+  repository: pennlabs/example-django
+  tag: latest
+  pullPolicy: IfNotPresent
+
+service:
+  port: 80
+
+ingress:
+  enabled: true
+  domain: example-django.upenn.club
```

Comment:
Again, we should just use a sample domain here.

Reply:
how is this not a sample domain?

Reply:
I was thinking something like example.pennlabs.org since we may not own upenn.club forever.

Reply:
We’re going  to own that domain for the rest of my life if Davis has anything to do with it

Reply:
We’re going  to own that domain for the rest of my life if Davis has anything to do with it